lib/rust/ast/generation/src/api.rs

```diff
@@ -229,55 +245,63 @@ impl Source {
     /// Generate API definition for the given AST definition.
     pub fn api(ast:&str) -> String {
         let input = syn::parse_str(ast).unwrap();
-        let file  = File::new("Ast", "org.enso.ast", input);
+        let file  = File::new("Ast","org.enso.ast",input);
         let api   = Self::new(file.into()).ast_api();
-
         api.to_string()
     }
 
     /// Create a new instance from `Collector`.
-    pub fn new(collector: PreprocessedFile) -> Self {
+    pub fn new(file:PreprocessedFile) -> Self {
         fn apply(typ:&Type, vars:&Map<&Name,&Type>) -> Type {
             if let Some(&typ) = vars.get(&typ.name) { typ.clone() } else {
                 let args = typ.args.iter().map(|typ| apply(typ,&vars)).collect();
-                Type::new(typ.name.clone(), vec![], args)
+                Type::new(typ.name.clone(),Vec::default(),args)
```

Comment:
This should be inferred, so you should be able to use bare `default()`. Specifying the type you're defaulting kinda removes the usefulness of using default over `vec![]`.